qemu/unicorn_common.h

```diff
@@ -91,6 +94,19 @@ static inline void target_page_init(struct uc_struct* uc)
     uc->target_page_align = TARGET_PAGE_SIZE - 1;
 }
 
+static uc_err uc_set_tlb(struct uc_struct *uc, int mode) {
+    switch (mode) {
+        case UC_TLB_VIRTUAL:
```

Comment:
I will prefer to setting `UC_TLB_CPU` by default since we want more compatibility for previous code, though they rely on the dirty or even wrong abstraction we provided before.

Reply:
This function is the implementation for `UC_CTL_TLB_TYPE`.

`UC_TLB_CPU` is set as default in uc.c:235, There is a check to prevent overriding a previous setting of the tlb_type.